src/utils/config.ts

```diff
@@ -0,0 +1,15 @@
+import { join } from 'path';
+import yaml from 'js-yaml';
+import { readFileSync, writeFileSync } from 'fs';
+
+export const APPLICATION_YML_RELATIVE_PATH = '../../compose-network/mirror-node/application.yml';
+
+export default function readApplicationYML() {
+    const propertiesFilePath = join(__dirname, APPLICATION_YML_RELATIVE_PATH);
+    const application = yaml.load(readFileSync(propertiesFilePath).toString()) as any;
+
+    return {
+        propertiesFilePath,
+        application
+    }
+}
```

Comment:
I like the idea, but this is only used in the debug state class, move this as a private method there. If we need it elsewhere, we can maybe add it to a service or something else, to keep the structure clean

Reply:
It's used on 2 occasions in the code. That is why I extracted it out, but I did not do the refactor as its not as part of this task. I'll do it in another MR. This one is complicated enough.